### PR TITLE
Add dnstap recordings for DVE-2017-0007, add updated information about  issue fixed

### PR DESCRIPTION
**Pull request by oerdnj**

### CODE REVIEW

File: 2017/DVE-2017-0007.md [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
-[RFC 7816](https://tools.ietf.org/html/rfc7816).  As an additional
-*bonus* the packets are malformed as there are extra 11 bytes at the
-end of the message and sets `AD` bit on in all DNS responses violating
-[RFC 4035](https://tools.ietf.org/html/rfc4035#section-3.1.6)
+`sr01.cdn.cra.cz.` and `sr02.cdn.cra.cz.` nameservers.  These
+authoritative nameservers fails to answer to `IN NS` queries by
+returning `RCODE=NOTIMPL`and thus breaking QNAME minimization [RFC
+7816](https://tools.ietf.org/html/rfc7816).
+
+There's additional breakagin in the DNS packets as they are malformed
```
Comment (jelu) [inline 2017/DVE-2017-0007.md]: Typo `breakagin` and is this not another DVE?

File: 2017/DVE-2017-0007.md [hunk 2]
```diff
@@ -3,13 +3,18 @@
...
+with extra 11 bytes at the end of the message when EDNS is used.
+
+The authoritative servers were also setting `AD` bit on in all DNS
```
Comment (jelu) [inline 2017/DVE-2017-0007.md]: This also, another DVE?

**Review by jelu — CHANGES_REQUESTED** [review]

**Comment by jelu** [comment]
@oerdnj ping?

**Comment by oerdnj** [comment]
Ok, I will split it into more DVEs soon..ish.

File: 2017/DVE-2017-0007.md (hunk 1 above)
Comment (oerdnj) [reply 2017/DVE-2017-0007.md]: This is not `DVE-2017-0012`

File: 2017/DVE-2017-0007.md (hunk 2 above)
Comment (oerdnj) [reply 2017/DVE-2017-0007.md]: This is now `DVE-2017-0013`

**Review by jelu — APPROVED** [review]

**Pull request merged**